tests/test_adding_contributor_views.py

```diff
@@ -546,445 +546,3 @@ def test_send_claim_email_before_throttle_expires(self):
-        preprint_source_tag, created = Tag.all_tags.get_or_create(name=provider_source_tag(self.preprint_with_source_tag.provider._id, 'preprint'), system=True)
-        self.project_with_source_tag.add_system_tag(osf_source_tag.name)
-        self.preprint_with_source_tag.add_system_tag(preprint_source_tag.name)
-        self.given_name = fake.name()
-        self.given_email = fake_email()
-        self.project_with_source_tag.add_unregistered_contributor(
-            fullname=self.given_name,
-            email=self.given_email,
-            auth=Auth(user=self.referrer)
-        )
-        self.preprint_with_source_tag.add_unregistered_contributor(
-            fullname=self.given_name,
-            email=self.given_email,
-            auth=Auth(user=self.referrer)
-        )
-        self.user = self.project.add_unregistered_contributor(
-            fullname=self.given_name,
-            email=self.given_email,
-            auth=Auth(user=self.referrer)
-        )
-        self.project.save()
-
-        self.mock_send_grid = start_mock_send_grid(self)
-
-    @mock.patch('website.project.views.contributor.send_claim_email')
-    def test_claim_user_already_registered_redirects_to_claim_user_registered(self, claim_email):
-        name = fake.name()
-        email = fake_email()
-
-        # project contributor adds an unregistered contributor (without an email) on public project
-        unregistered_user = self.project.add_unregistered_contributor(
-            fullname=name,
-            email=None,
-            auth=Auth(user=self.referrer)
-        )
-        assert unregistered_user in self.project.contributors
-
-        # unregistered user comes along and claims themselves on the public project, entering an email
-        invite_url = self.project.api_url_for('claim_user_post', uid='undefined')
-        self.app.post(invite_url, json={
-            'pk': unregistered_user._primary_key,
-            'value': email
-        })
-        assert claim_email.call_count == 1
-
-        # set unregistered record email since we are mocking send_claim_email()
-        unclaimed_record = unregistered_user.get_unclaimed_record(self.project._primary_key)
-        unclaimed_record.update({'email': email})
-        unregistered_user.save()
-
-        # unregistered user then goes and makes an account with same email, before claiming themselves as contributor
-        UserFactory(username=email, fullname=name)
-
-        # claim link for the now registered email is accessed while not logged in
-        token = unregistered_user.get_unclaimed_record(self.project._primary_key)['token']
-        claim_url = f'/legacy/user/{unregistered_user._id}/{self.project._id}/claim/?token={token}'
-        res = self.app.get(claim_url)
-
-        # should redirect to 'claim_user_registered' view
-        claim_registered_url = f'/legacy/user/{unregistered_user._id}/{self.project._id}/claim/verify/{token}/'
```

Comment:
All conflicts in `TestClaimViews` seems to be caused by the `legacy` change. I assume we have moved the tests to somewhere else and have already fixed the test in other PRs. This is why we just need to remove everything here?